packages/wonder-blocks-banner/src/components/__docs__/banner.stories.js

```diff
@@ -393,6 +393,83 @@ WithDismissal.parameters = {
     },
 };
 
+export const WithCustomAction: StoryComponentType = () => (
+    <Banner
+        text="some text"
+        layout="floating"
+        actions={[
+            {
+                type: "custom",
+                node: (
+                    <Button
+                        kind="tertiary"
+                        size="small"
+                        onClick={() => {}}
+                        spinner={true}
+                    >
+                        Spinner Button
+                    </Button>
+                ),
+            },
+        ]}
+    />
+);
+
+WithCustomAction.parameters = {
+    docs: {
+        storyDescription: `There are a number of other props that
+            Buttons and Links may have that are not currently supported by
+            the \`actions\` prop in Banner. These would require the use of
+            custom actions. If it absolutely necessary to have a custom
```

Comment:
**suggestion:** Even that you mention here that this can be done (if necessary), I feel that it would be important to state as the first phrase that this is discouraged (unless it's the last resource - not sure about the exact wording about that). wdyt?

Reply:
How about this?

<img width="1058" alt="image" src="https://user-images.githubusercontent.com/13231763/205778890-4df274f5-18bb-4db1-b235-716056b7e767.png">